src/apps/concepts/components/AddConceptToDictionaryIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    Fab,
+    Tooltip
+} from "@material-ui/core";
+import {APIConcept} from "../types";
+
+import {Add as AddIcon} from "@material-ui/icons";
+
+
+
+interface AddConceptToDictionaryIconProps {
+    dictionaryToAddTo:string;
+    canAddConcept:string;
+    row:APIConcept|undefined ;
+    addConceptsToDictionary: Function;
+   
+}
+
+
+    export function AddConceptToDictionaryIcon(props: AddConceptToDictionaryIconProps) {
+        const {
+          dictionaryToAddTo,
+          canAddConcept,
+          row,
+          addConceptsToDictionary
+        } = props;
+    return (canAddConcept=="false"  || dictionaryToAddTo==null || dictionaryToAddTo==undefined)? (null):
+        (
+            <Tooltip title="Add concept to Dictionary" data-testid="addConceptToDictionaryIcon">
+                <Fab onClick={() => {
+                                if (addConceptsToDictionary){ 
```

Comment:
Do we need this `if` check?

Reply:
@ibacher see [here](https://github.com/openmrs/openmrs-ocl-client/blob/2dcb3ceebc21dea96044e5eddd7c333bd506bf6d/src/apps/concepts/components/ConceptsActionMenu.tsx#L27)
it is I tried to replicate it